libbeat/processors/actions/add_fields.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/processors"
 	"github.com/elastic/beats/libbeat/processors/checks"
+	jsProcessor "github.com/elastic/beats/libbeat/processors/script/javascript/module/processor"
```

Comment:
Nit. Not sure if there is a convention about that but package names use to be all lowercased, maybe we could use `jsprocessor` for this.
```suggestion
	jsprocessor "github.com/elastic/beats/libbeat/processors/script/javascript/module/processor"
```